templates/index.html

```diff
@@ -3,34 +3,64 @@
   <head>
     <title>Play Nim</title>
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <style>
+      body {
+        background-color: #EEEEEE;
+      }
+      div {
+        color: #555555;
+        padding: 10px 10px 10px 10px;
+      }
+    </style>
   </head>
   <body>
-  <center>
 
-    <h1>So it begins...</h1>
+    <div>
+      <h1>So it begins...</h1>
+    </div>
 
-        <h3>
-        Enter Minimum stones per pile <br><input name="desired_min" type="text" size="5"> <br><br>
-        Enter Maximum stones per pile <br><input name="desired_max" type="text" size="5" > <br><br>
-        Enter Number of piles <br><input name="desired_piles" type="text" size="5"> <br><br>
-        </h3>
+    <div style="float: left;"><h1 id="pile0" ></h1></div>
+    <div style="float: left;"><h1 id="pile1" ></h1></div>
+    <div style="float: left;"><h1 id="pile2" ></h1></div>
+    <div style="float: left;"><h1 id="pile3" ></h1></div>
+    <div style="float: left;"><h1 id="pile4" ></h1></div>
 
-    <table>
-        <tr>
-        <td><div id="pile0"></div></td>
-        <td><div id="pile1"></div></td>
-        <td><div id="pile2"></div></td>
-        <td><div id="pile3"></div></td>
-        <td><div id="pile4"></div></td>
-        </tr>
-    </table>
+    <div style="clear: both;">
+      <button onclick="updateGameState()">Make Move</button>
+      <button onclick="newGame()">New Game</button>
+    </div>
 
-    <br><br>
+    <div>
+      <select id="piles">
+        <option value=3>3</option>
+        <option value=4>4</option>
+        <option value=5>5</option>
+      </select>
```

Comment:
Doing what? The front end has a finite number of elements that we can populate with piles. I can make any number of them. I just chose 3 to 5 because that's what I generally end up playing in real life situations. Were you asking for a lower min, higher max, or to let them choose any number?

Reply:
Let us choose any number. If later on we realize that it doesn't make sense and we need to limit it, then sure. But for now we don't need to limit ourselves.

Reply:
Hmmm... The jQuery is updating a finite number of HTML elements. It locates them by ID. Should I add an infinite number of HTML tags? See the `id="pile0"` in `<div style="float: left;"><h1 id="pile0" ></h1></div>`.

Reply:
I have no idea how to do it, but there has to be a way to make it work with regular text boxes.

Reply:
This is a bigger ask than you realize. We can always `curl` if we want to test extreme cases. Let's file a ticket for this work and settle on a min and a max for now.

Reply:
Disregard previous comment. I've implemented this.

Reply:
As I just mentioned on Slack, I was only asking you to revert the min/max/piles thing so they were text boxes. Changing `updatePage` wasn't necessary but I'm glad it's done, now I believe we have a fully functional UI thanks to you :tada: